### PR TITLE
Create a component to add subcategories

### PR DESCRIPTION
**Pull request by anjula-sack**
## Purpose

- The purpose of this PR is to fix #65 
## Goals

- Adding maintain functionalities to admin dashboard to maintain academix
## Approach

- Create a component to add subcategories
- Edit AddItem component according to the new payload
- Remove redundant scope from main style.css
sub

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: frontend/app/src/scenes/Home/scenes/AddSubCategory/index.tsx [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+            <Title>Add a new Subcategory</Title>
+            <Row>
+              <Col md={12}>
+                <Spin tip="Loading..." spinning={this.state.isLoading}>
+                  <Form size="large" onFinish={this.onFinish}>
+                    <Title level={3}>Categories</Title>
+                    <Text>Select Categories for the subcategory</Text>
+                    <Form.Item name="category" className={mainStyles.formItem}>
+                      <Select
+                        style={{ width: '100%' }}
```
Comment (jayasanka-sack) [inline frontend/app/src/scenes/Home/scenes/AddSubCategory/index.tsx:101]: Remove inline styles

Comment (anjula-sack) [reply frontend/app/src/scenes/Home/scenes/AddSubCategory/index.tsx]: done

File: frontend/app/src/scenes/Home/scenes/AddItem/index.tsx [hunk 2]
```diff
@@ -74,7 +69,7 @@ class AddItem extends React.Component<RouteComponentProps, AddItemStateProps> {
             message: 'Success!',
             description: 'Successfully Created an Item',
           });
-          this.props.history.push(`${res.data.id}/edit`);
+          // this.props.history.push(`${res.data.id}/edit`);
```
Comment (jayasanka-sack) [inline frontend/app/src/scenes/Home/scenes/AddItem/index.tsx:72]: Do not add commented codes

File: frontend/app/src/scenes/Home/scenes/AddSubCategory/index.tsx [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+    return (
+      <div>
+        <Row className={mainStyles.innerContent}>
+          <Col lg={24} xl={{ span: 20, offset: 2 }}>
+            <Title>Add a new Subcategory</Title>
+            <Row>
+              <Col md={12}>
+                <Spin tip="Loading..." spinning={this.state.isLoading}>
+                  <Form size="large" onFinish={this.onFinish}>
+                    <Title level={3}>Categories</Title>
```
Comment (jayasanka-sack) [inline frontend/app/src/scenes/Home/scenes/AddSubCategory/index.tsx:97]: It should be "category"

File: frontend/app/src/scenes/Home/scenes/AddSubCategory/index.tsx [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+                    <Text>Select Categories for the subcategory</Text>
```
Comment (jayasanka-sack) [inline frontend/app/src/scenes/Home/scenes/AddSubCategory/index.tsx:98]: "category"

File: frontend/app/src/scenes/Home/styles.css [hunk 5]
```diff
@@ -39,6 +39,10 @@
     margin-top: 12px;
 }
 
+:local .formSelect {
```
Comment (jayasanka-sack) [inline frontend/app/src/scenes/Home/styles.css:42]: remove `:local`

Comment (anjula-sack) [reply frontend/app/src/scenes/Home/styles.css:42]: from the whole sheet?

Comment (jayasanka-sack) [reply frontend/app/src/scenes/Home/styles.css:42]: yeah it's better to remove all. We don't need it. It's `:local` by default.

Comment (anjula-sack) [reply frontend/app/src/scenes/Home/styles.css:42]: okay

File: frontend/app/src/scenes/Home/scenes/AddSubCategory/index.tsx (hunk 3 above)
Comment (anjula-sack) [reply frontend/app/src/scenes/Home/scenes/AddSubCategory/index.tsx]: ok

File: frontend/app/src/scenes/Home/scenes/AddItem/index.tsx (hunk 2 above)
Comment (anjula-sack) [reply frontend/app/src/scenes/Home/scenes/AddItem/index.tsx]: ok

File: frontend/app/src/scenes/Home/scenes/AddSubCategory/index.tsx [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+      <div>
+        <Row className={mainStyles.innerContent}>
+          <Col lg={24} xl={{ span: 20, offset: 2 }}>
+            <Title>Add a new Subcategory</Title>
+            <Row>
+              <Col md={12}>
+                <Spin tip="Loading..." spinning={this.state.isLoading}>
+                  <Form size="large" onFinish={this.onFinish}>
+                    <Title level={3}>Category</Title>
+                    <Text>Select category for the subcategory</Text>
```
Comment (jayasanka-sack) [inline frontend/app/src/scenes/Home/scenes/AddSubCategory/index.tsx:104]: Shall we replace this with "Select a category"

Comment (anjula-sack) [reply frontend/app/src/scenes/Home/scenes/AddSubCategory/index.tsx:104]: ok

**Pull request merged**